wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,166 @@
+
+### During the tournament
+
+If a tournament satisfies the criteria outlined above, it must also abide by the following practices during play:
+
+- Tournament organizers will be expected to monitor the matches of their tournament for toxic behavior or anything that goes against the expectations laid out above, and report it to the osu! Support Team.
+- Tournament organizers who spot potentially suspicious play during their tournament are **single-handedly responsible for reporting such incidents to the osu! Support Team** via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+- All multiplayer games relevant to the tournament must be created with the ```!mp make``` command, so that they do not expire.
+- All matches undertaken for the tournament must be recorded and their history made publicly available on the original forum post for the tournament in a clearly marked link in normal size font or larger.
+- In the event that a player becomes restricted during the course of the tournament, you may need to replay certain matches where they were involved in order to preserve the competitive integrity of the tournament.
+    - Organizers should contact the Support Team to receive information and guidance on these (hopefully rare) cases.
+
+### After the tournament
+
+Once the tournament has concluded, the organizer will need to submit the following to the osu! Support Team:
+
+- A list of applicable badge recipients (aka: the winners of your tournament)
+- A link to the publicly viewable catalogue of your tournament's match history
+
+LAN tournaments are also required to add links to VODs and a copy of the bracket of the entire tournament in order to be considered for a badge prize.
+
+## Frequently Asked Questions (FAQ)
+
+### Why does a tournament have to run at least twice or more in order to qualify for profile badges without experienced staff present?
+
+The second iteration of a tournament is often free of a lot of the organizational issues of the first tournament, and has established a solid workflow and staff involved in its production. They are far less likely to peter out or be subject to questionable choices. The tournament is also far more identifiable if it runs more than just once.
+
+For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.
+
+### What is considered 'staff' as far as tournament management goes?
+
+We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
+
+Commentators are not considered by us as staff, and may play or interact in tournaments freely so long as they are not directly involved any managerial aspect of running the tournament or violate anything mentioned above.
```

Comment:
Bit nitpicky, but maybe mention "streamers" too?

Reply:
Streamers dont affect the outcome of a match, as refs and commentators could do.

Reply:
How would a commentator be able to affect the outcome of a match where a streamer would not?

Reply:
Isn't PooN just saying to mention streamers in the line about commentators?

Reply:
> Streamers dont affect the outcome of a match, as refs and commentators could do.

I don't think streamers and commentators affect the outcome of a match.

To clarify:  
In commonly-structured tournaments, streamers are another position players of a badged tournament can apply for. Thus, I suggest to mention it alongside commentators.

Reply:
Yeah, i should made it clearer too, streamers dont directly intervene on matches, commentators dont have issues to participate but there were instances in the past that commentators hated a certain player and pointed bad things of him on stream, I think this goes more on tournament staff behaviour but depending if the player is listening to the stream that could affect their tournament performance.
Back to the point, no issues on making it clear that commentators and streamers shouldnt have issues on playing.